packages/cyberstorm/src/components/CustomDropDown.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { ReactNode } from "react";
+import styles from "./componentStyles/CustomDropDown.module.css";
+import { Button } from "./Button";
+import { faChevronDown } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import * as DropdownMenu from "@radix-ui/react-dropdown-menu";
+import { SelectItemProps } from "./CustomSelectItem";
+
+type DropDownProps = {
+  colorScheme?: "default" | "defaultDark" | "primary";
+  defaultOpen?: boolean;
+  icon?: ReactNode;
+  content?: ReactNode[];
+  label?: string;
+};
+
+export const CustomDropDown: React.FC<DropDownProps> = (props) => {
+  const { colorScheme, defaultOpen, icon, content, label } = props;
+
+  return (
+    <div className={styles.root}>
+      <DropdownMenu.Root defaultOpen={defaultOpen}>
+        <DropdownMenu.Trigger asChild>
+          <div>
+            <Button label={label} colorScheme={colorScheme} rightIcon={icon} />
+          </div>
+        </DropdownMenu.Trigger>
+
+        <DropdownMenu.Portal>
+          <DropdownMenu.Content
+            align="start"
+            sideOffset={8}
+            className={`${styles.content} ${getContentStyle(colorScheme)}`}
+          >
+            <DropdownMenu.Group className={styles.group}>
```

Comment:
Is nesting everything inside a group by default even if the items don't belong in a group by nature intentional? This will also prevent any additional grouping of items.

Reply:
True, removed the unnecessary layer of grouping.